e2e/e2eutils/setup.go

```diff
@@ -115,7 +115,7 @@ func MakeDeployParams(t require.TestingT, tp *TestParams) *DeployParams {
 		DeploymentWaitConfirmations: 1,
 		ValidatorRewardScalar:       5000,
 
-		SecurityCouncilTokenOwner:    addresses.SysCfgOwner,
+		ProxyAdminOwner:              addresses.SysCfgOwner,
```

Comment:
IMO, using both `ProxyAdminOwner` and `SysCfgOwner` could be confusing. 
Why don't we change `SysCfgOwner` to `ProxyAdminOwner` as well?

Reply:
I added one more commit applying it. Please check it.